readthedocs/oauth/services/base.py

```diff
@@ -177,15 +188,16 @@ def update_webhook(self, project, integration):
 
     @classmethod
     def is_project_service(cls, project):
-        """Determine if this is the service the project is using
+        """
+        Determine if this is the service the project is using.
 
         .. note::
+
             This should be deprecated in favor of attaching the
-            :py:class:`RemoteRepository` to the project instance. This is a slight
-            improvement on the legacy check for webhooks
+            :py:class:`RemoteRepository` to the project instance. This is a
+            slight improvement on the legacy check for webhooks
         """
         # TODO Replace this check by keying project to remote repos
         return (
             cls.url_pattern is not None and
-            cls.url_pattern.search(project.repo) is not None
-        )
+            cls.url_pattern.search(project.repo) is not None)
```

Comment:
All these are linting changes, I think? This is a bit of what I was worried about w/ the auto-linting, making it hard to catch real changes in the massive linting changes.

Reply:
Yes, they are linting changes.

I try to make these changes in separated commits but sometimes it's complicated.

I don't know what's the best way for this. I think we can make a *huge* PR with all the automcatic linting changes and fix the specific one by hand after merging it. That will make PR with linting changes smaller in the future.

But I don't know what's best, honestly (I'm used to review PR with a lot of linting changes because we followed the same as I'm doing here but...)

Reply:
We discussed this a little. I don't think large commit is the answer yet, this shouldn't be priority. We can however be more careful in making PRs where autolint is a separate commit. This should be more easy to review, but requires some careful git rebasing depending on your workflow (if you are commiting autolint changes with your logic changes)

Or just a separate PR, whatever is easier to manage for your own sake.